Dockerfile.psql.development

```diff
@@ -0,0 +1,62 @@
+#base image
+FROM ubuntu:16.04
+MAINTAINER OriginTrail
+LABEL maintainer="OriginTrail"
+ARG targetEnvironment=development
+
+ENV NODE_ENV=$targetEnvironment
+ENV GRANAX_USE_SYSTEM_TOR=1
+ENV OT_NODE_DISTRIBUTION=docker
+ENV DB_TYPE=psql
+
+RUN apt-get -qq update && apt-get -qq -y install curl
+RUN curl -sL https://deb.nodesource.com/setup_9.x |  bash -
+RUN apt-get -qq update && apt-get -qq -y install wget apt-transport-https software-properties-common build-essential git nodejs sqlite unzip nano
+RUN add-apt-repository -y ppa:ethereum/ethereum && apt-get -qq update && apt-get install -y -qq ethereum geth
+#ArangoDB
+ADD testnet/install-arango.sh /install-arango.sh
+RUN ["chmod", "+x", "/install-arango.sh"]
+RUN /install-arango.sh
+
+RUN export LC_ALL=C
+RUN apt-get install -y libpq-dev
+RUN apt-get update && apt install -y -qq supervisor
+RUN mkdir -p /var/log/supervisor
+COPY testnet/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+RUN apt-get -y install nano wget
+
+#PostgresDB
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" |tee  /etc/apt/sources.list.d/pgdg.list
+RUN apt-get update && apt-get install -y python-software-properties software-properties-common postgresql-12 postgresql-client-12 postgresql-contrib-12
+USER postgres
+RUN    /etc/init.d/postgresql start &&\
+    psql --command "CREATE USER root WITH SUPERUSER PASSWORD 'origintrail'" &&\
```

Comment:
We should avoid using default password for postgresql superuser

Reply:
We need to set initial password when building docker image. When we run container for the first time, migrations will be executed. In migration we are updating root and user password from configuration.